src/messaging/ExchangeContext.cpp

```diff
@@ -113,7 +113,8 @@ CHIP_ERROR ExchangeContext::SendMessage(Protocols::Id protocolId, uint8_t msgTyp
 CHIP_ERROR ExchangeContext::SendMessageImpl(Protocols::Id protocolId, uint8_t msgType, PacketBufferHandle msgBuf,
                                             const SendFlags & sendFlags, Transport::PeerConnectionState * state)
 {
-    CHIP_ERROR err = CHIP_NO_ERROR;
+    CHIP_ERROR err                     = CHIP_NO_ERROR;
+    bool reliableTransmissionRequested = true;
```

Comment:
please declare the bool when it is first use. declaring bools at the top  of the function is C-style and only forced by the 'goto exit' macros.

Reply:
ack